react-native-sdk/android/src/main/java/org/jitsi/meet/sdk/JMOngoingConferenceModule.java

```diff
@@ -0,0 +1,69 @@
+package org.jitsi.meet.sdk;
+
+import android.app.Notification;
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.module.annotations.ReactModule;
+
+
+@ReactModule(name = JMOngoingConferenceModule.NAME)
+class JMOngoingConferenceModule
+    extends ReactContextBaseJavaModule {
+
+    public static final String NAME = "JMOngoingConference";
+
+    private static JMOngoingConferenceModule jmOngoingConferenceModuleInstance;
+
+    public JMOngoingConferenceModule(ReactApplicationContext reactContext) {
+        super(reactContext);
+        jmOngoingConferenceModuleInstance = this;
+    }
+
+    static JMOngoingConferenceModule getInstance() {
+        return jmOngoingConferenceModuleInstance;
+    }
+
+    Notification build() {
```

Comment:
We should be able to pass the right parameters to `launch` so this is not necessary. Why did you need to add this here? Feels kinda out of place since the notification is controlled by the service, not this.